### ISSUE TITLE
I want to create a RLHF backend/frontend for labelling<=>training<=>correcting error loop.

### ISSUE THREAD

**Issue by hemangjoshi37a** [heartexlabs/label-studio-ml-backend#233]
If anyone has any lead on this please let me know. also anyone want to collaborate on this direction please let me know.

**Comment by makseq** [comment]
Sound like active learning? 

https://docs.heartex.com/guide/active_learning.html

https://www.youtube.com/watch?v=8EO4vOw1MZc

**Comment by hemangjoshi37a** [comment]
@makseq While active learning is good but RLHF is quite different than that becuase it implements Reignforcement Learning for optimization of the model. All in all if you know what is RLHF it is quite different than active learning.

**Comment by makseq** [comment]
Yes, I know, but I expect to see your workflow in LS to achieve it. Seems you need Accept/Reject actions for your annotations?

**Comment by hemangjoshi37a** [comment]
Yes the RLHF can be done in multiple ways. You can have yes no type or ranking type.

**Comment by hemangjoshi37a** [comment]
Basically what I propose is the have a generalized RLHF model that goes at the output side of any model and instead of having supervised training we can have unsupervised training that can be supervised by the reinforcement model.

**Comment by makseq** [comment]
Maybe this repo will be helpful for you: https://github.com/heartexlabs/label-studio-RLHF/

**Comment by hemangjoshi37a** [comment]
@makseq maybe it is a private repo. giving me 404 error

**Comment by makseq** [comment]
Sorry, could you please check this one:
https://github.com/heartexlabs/RLHF
